frb/analysis/dmhalos_photoz.py

```diff
@@ -0,0 +1,615 @@
+import numpy as np, os, glob
```

Comment:
I feel like this module might go best in `halos`
Thoughts?

Reply:
Makes sense. I shall move it there.